pavelib/assets.py

```diff
@@ -292,6 +301,27 @@ def compile_templated_sass(systems, settings):
         print("\t\tFinished preprocessing {} assets.".format(system))
 
 
+def process_npm_assets():
```

Comment:
Is this something that we need to do in production as well? Is there a corresponding configuration PR?